package.json

```diff
@@ -18,10 +18,8 @@
   },
   "homepage": "https://github.com/FACG4/PWD-helpers#readme",
   "dependencies": {
-    "brreg": "^3.0.0",
     "docxtemplater": "^3.6.6",
     "env2": "^2.2.0",
-    "eslint": "^4.19.1",
```

Comment:
move this to devdependencies.

Reply:
which one exactly?! the eslint is already removed from here!